crates/stellar-scaffold-cli/README.md

```diff
@@ -80,6 +80,18 @@ The build process ensures:
 - TypeScript client generation for frontend integration
 - Contract state verification and updates
 
+### Setting contract metadata
+
+Contract metadata is set when running `stellar scaffold build` and adds the following fields from your contract's Cargo.toml to the contract metadata.
+
+| Metadata   | Cargo.toml Key (under `package`) | Description                                      |
+|------------|----------------|--------------------------------------------------|
+| `name`     | `name`         | The name of the contract                         |
+| `binver`   | `version`      | The version of the WASM bytecode of the contract                      |
+| `authors`  | `authors`      | The author(s) of the contract, often with an email  |
+| `home_domain` | `homepage`  | The relevant domain to relate to this contract   |
+| `source_repo` | `repository` | The source repository URL for the contract      |
+
```

Comment:
`name` and `authors` belong to this section. 

Everything else is non-standard and need to move their own custom section, with the SEP to define this mapping as names do not intuitively map for a new developer.